components/x-interaction/src/concerns/serialiser.js

```diff
@@ -1,6 +1,7 @@
 import { h, render } from '@financial-times/x-engine'
 import getComponentName from './get-component-name'
```

Comment:
`getComponentName` is used below when adding serialisation data, but it still relies on using `.displayName`/`.name`. it should be moved into `register-component.js` and retrieve the name using the `x-interaction` name property